rotkehlchen/tests/api/test_history_base_entry.py

```diff
@@ -1124,3 +1126,123 @@ def test_add_edit_evm_swap_events(rotkehlchen_api_server: 'APIServer') -> None:
         'product': 'pool',
         'address': '0xb5d85CBf7cB3EE0D56b3bB207D5Fc4B82f43F511',
     }
+
+
+def test_multi_grouping(rotkehlchen_api_server: 'APIServer') -> None:
+    """Test that events are properly grouped into sub-lists
+    when they are serialized for the api.
+    """
```

Comment:
can you write in the docstring a summary of what are the events? Otherwise I need to go and read them and match them to understand what is a swap and so on.